src/main/java/org/dasein/cloud/network/VPN.java

```diff
@@ -113,4 +105,12 @@ public void setProviderVlanIds(String[] providerVlanIds) {
     public String toString() {
         return providerVpnId;
     }
+
+    public String getProviderVpnIP() {
+        return providerVpnIP;
```

Comment:
`VpnIP` reminds me of the infamous `XmlHTTPRequest`. Make up your mind, but I suggest we move towards the correct camel-casing, e.g. `VpnIp`.